pandas/core/generic.py

```diff
@@ -2806,6 +2806,11 @@ def drop(self, labels=None, axis=0, index=None, columns=None, level=None,
         -------
         dropped : type of caller
 
+        Raises
+        ------
+        KeyError
+            * If labels are not found in the selected axis
```

Comment:
Don't think you need the `*` here. And maybe clarify that it raises if *none* of the labels are found.